contracts/sdk/indexer/db.ts

```diff
@@ -83,6 +100,35 @@ export class NFTDatabaseConnection {
     }
   }
 
+  /// Can be used to kick of gapfill for all leafnodes in db
+  async findMostRecentNodes() {
+    return await this.connection.run(
+      `
+      SELECT * FROM merkle 
+      WHERE seq in 
+        (SELECT seq FROM 
+          (SELECT node_idx, MAX(seq) AS seq 
+          FROM merkle 
+          WHERE level = 0 group by node_idx
+        ));
+      `
+    );
+  }
+
+  async dropMostRecentNodes() {
+    return await this.connection.run(
+      `
+      DELETE FROM merkle
+      WHERE seq in 
+        (SELECT seq FROM 
+          (SELECT node_idx, MAX(seq) AS seq 
+          FROM merkle 
+          WHERE level = 0 group by node_idx
+        ));
+      `
+    );
+  }
```

Comment:
This is used for testing the backfill scripts.

Reply:
Let's not commit this one lol. There's literally no use case except for testing

Reply:
Okay fair point, I'd rather add to a readme